recipes/stopt/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "stopt" %}
+{% set version = "5.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://gitlab.com/stochastic-control/StOpt
+  git_rev: "v{{ version }}" # (Defaults to "HEAD")
+  # git_depth: 1 # (Defaults to -1/not shallow)
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake >=3.15
+    - make
+    - pkgconfig  # [unix]
+  host:
+    - boost  # TODO: revise to libboost-devel in the future
+    - libboost-mpi
+    - mpich  # [unix]
+    - msmpi  # [win]
+    - _openmp_mutex  # [linux]
+    - llvm-openmp  # [osx or win]
+    - eigen
+    - coin-or-utils
+    - coin-or-clp
+    - coin-or-osi
+    - bzip2
+    - zlib
+  run:
+    - libboost
+    - libboost-mpi
+    - mpich  # [unix]
+    - msmpi  # [win]
+    - _openmp_mutex  # [linux]
+    - llvm-openmp  # [osx or win]
+    - eigen
+    - bzip2
+    - zlib
```

Comment:
You can probably omit all of these as they will come in via `run_exports`
```suggestion
```

Reply:
Thanks for the advise, I'll experiment locally and commit when tested

Reply:
I've added run_exports and kept libboost that was needed